spec/js/ClickCounter.spec.js

```diff
@@ -0,0 +1,51 @@
+describe("ClickCounter is a class that counts clicks and turns them into donuts, a currency to enhance the ability to buy upgrades to your clicking button.", () => {
+    let underTest;
+    beforeEach(() => {
+        underTest = new ClickCounter();
+    });
+    describe("FEATURE : Have a way to count clicks, recording them as donuts.", () => {
+        for (let i = 0; i < 10; i++) {
+            it("Should record " + i + " click(s), increasing the donut count to " + i + ".", () => {
+                expectOneToOneClickToDonutCount(i, underTest);
+            });
+        }
+    });
+    /*
+    > As a user, I want to be able to purchase _Auto Clickers_, so that I don't have to keep clicking a button all day.0
+    - Can retrieve a _Clicking Companion_ count.
+    - Can add to the _Clicking Companion_ count.
+    - Subtract the amount of the _Clicking Companion_ cost from your click count.
+    */
```

Comment:
You can move the content of this comment into the description of the unit test to make a more 'living' test suite.

Reply:
Ok, updated in a future commit.

Reply:
We need to talk about this,